katas/langtonant/maria/antolangt.js

```diff
@@ -0,0 +1,180 @@
+
+function Grid(n, m) {
+    this.n = n;
+    this.m = m;
+    var matriz = new Array();
+    for (let i = 0; i <= n - 1; i++) {
+        matriz[i] = new Array();
+        for (let j = 0; j <= m - 1; j++) {
+            matriz[i][j] = 0;
+        }
+    }
+    this.cargar = function() {
+        for (let i = 0; i <= n - 1; i++) {
+            for (let j = 0; j <= m - 1; j++) {
+                matriz[i][j] = this.random();
+            }
+        }
+    };
+
+    this.random = function() {
+        let randomN = Math.random();
+        if (randomN < 0.2) {
+            return 1;
+        }
+        return 0;
+    };
+    this.imprimir = function() {
+        for (let elemento in matriz) {
+            console.log(elemento + '=' + matriz[elemento]);
+        }
+    };
+
+    this.valor = function(n, m) {
+        return matriz[n][m];
+    };
+
+    this.setValor = function(n, m, v) {
+        matriz[n][m] = v;
+    };
+    this.setAnt = function(a, c) {
+        matriz[a][c] = '&';
+    };
+
+    /* this.rigth=function(direction){
```

Comment:
Lets remove commented code, or complete its functionality if it is not completed